mypy/find_sources.py

```diff
@@ -19,53 +20,76 @@ class InvalidSourceList(Exception):
     """Exception indicating a problem in the list of sources given to mypy."""
 
 
-def create_source_list(files: Sequence[str], options: Options,
-                       fscache: Optional[FileSystemCache] = None,
-                       allow_empty_dir: bool = False) -> List[BuildSource]:
+def partition(
+    pred: Callable[[str], bool], iterable: Iterable[str]
+) -> Tuple[Iterable[str], Iterable[str]]:
```

Comment:
We don't use this style for multi-line signatures. The original signature conforms to our style.